src/api/operator/operatorApi.ts

```diff
@@ -163,15 +163,26 @@ export async function getOrder(params: GetOrderParams): Promise<RawOrder> {
 
   const queryString = `/orders/${orderId}`
 
-  const response = await _get(networkId, queryString)
+  let response
+
+  try {
+    response = await _get(networkId, queryString)
+  } catch (e) {
+    const msg = `Failed to fetch ${queryString}`
+    console.error(msg, e.message)
+    throw new Error(msg)
+  }
 
   if (!response.ok) {
     throw new Error(`Request failed: [${response.status}] ${await response.text()}`)
```

Comment:
This await may also fail and will be uncaught but also probably fine

Reply:
Yeah... that's true.
Will address it in another PR.

Reply:
https://github.com/gnosis/gp-ui/pull/89#discussion_r566676968